site/data/i18n/en.json

```diff
@@ -186,13 +186,13 @@
 	"menu-footer-about-military-text": "Public service and military",
 	"menu-footer-about-military-link": "https://www.canada.ca/en/government/publicservice.html",
 	"menu-footer-about-news-text": "News",
-	"menu-footer-about-news-link": "http://news.gc.ca/",
```

Comment:
This needs to be updated yet again.  Going to news.gc.ca redirects to news.gc.ca/web/index-en.do, but then there's a modal alert that says the the page has been moved:

> You will now be redirected to the Newsroom of Canada.ca.
> If you are not redirected in the next 20 seconds, please visit https://www.canada.ca/en/news.html